bika/lims/profiles/default/workflows/bika_analysis_workflow/definition.xml

```diff
@@ -150,7 +151,11 @@
       <permission-role>LabManager</permission-role>
       <permission-role>Manager</permission-role>
     </permission-map>
-    <permission-map name="BIKA: Edit Field Results" acquired="True">
+    <permission-map name="BIKA: Edit Field Results" acquired="False">
+      <permission-role>Analyst</permission-role>
+      <permission-role>LabManager</permission-role>
+      <permission-role>Manager</permission-role>
+      <permission-role>Sampler</permission-role>
```

Comment:
Assuming that sample is received after the sample collection has taken place and the field results must be entered during the collecting process, no one should be able to edit those "field results" (except Sampler and Labman) once a Sample has been received.